### DISCUSSION TITLE
Where is release openssl-3.1.3?

### DISCUSSION THREAD

**Discussion by userdocs** [openssl/openssl#22152] [Q&A] [answered]
When tag [openssl-3.1.3](https://github.com/openssl/openssl/releases/tag/openssl-3.1.3) was pushed the corresponding release was not. Perhaps an oversight?

https://www.openssl.org

> 19-Sep-2023 | OpenSSL 3.1.3 is now available, including bug and security fixes

Current latest release is [openssl-3.1.2](https://github.com/openssl/openssl/releases/tag/openssl-3.1.2) - <https://github.com/openssl/openssl/releases/latest>

**Comment by mattcaswell** [comment, marked as answer]
It's there now.

**Reply by userdocs** [reply]
Thanks.
